pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/zookeeper/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: layout.pug
+navigationTitle: ZooKeeper
+title: ZooKeeper
+menuWeight: 5
+excerpt: Deploying ZooKeeper in a project
+---
+
+## Getting Started
+
+To get started with creating ZooKeeper clusters in your project namespace, you first need to deploy the [ZooKeeper operator](../../../../../workspaces/applications/catalog-applications/zookeeper-operator/) in the workspace where the project exists.
+
+Once you have the ZooKeeper operator deployed, you can create ZooKeeper Clusters by applying a `ZookeeperCluster` custom resource in a project's namespace.
+
+A [Helm chart](https://github.com/pravega/zookeeper-operator/tree/master/charts/zookeeper) exists in the ZooKeeper operator repository that can assist with deploying ZooKeeper clusters.
+
+### Example Deployment
+
+1.  Set the `PROJECT_NAMESPACE` environment variable to the name of your project’s namespace:
+
+    ```bash
+    export PROJECT_NAMESPACE=<project namespace>
+    ```
+
+1.  Create a ZooKeeper Cluster custom resource in your project namespace
+
+    ```yaml
+    kubectl apply -f - <<EOF
+    apiVersion: zookeeper.pravega.io/v1beta1
+    kind: ZookeeperCluster
+    metadata:
+      name: zookeeper
+      namespace: ${PROJECT_NAMESPACE}
+    spec:
+      replicas: 1
+    EOF
+    ```
+
```

Comment:
I think it would be really useful to have an example here that shows what this results in and how to check that it worked.  

For example, The operator documentation page has this very nice example:   https://github.com/pravega/zookeeper-operator#manual-deployment-1

Could we possibly duplicate some of that here?   I think it would make this documentation much more complete and usable.  

At the very minimum, can we please include the resources section (and links) from the workspace page on this page also? I know it's a bit redundant, but I think this page is where a developer would be looking for more information on how to configure zookeeper, how to access it,  and how to tell if it deployed, and it's a bit annoying to have to go back to the operator install page (which would be a one time thing per cluster, right?)

Reply:
Just something simple like, @cneth ? : 

Check your ZooKeeper cluster using kubectl: 

kubectl get zk

Reply:
I took a stab at that in cfa2be8cb5c4e4a2631c3ad1c19923ea6495cc86, I can always add more like describing the cluster and viewing all the associated resources!

Reply:
This looks great.  TY!